src/leader_cron_task.erl

```diff
@@ -495,6 +495,17 @@ extract_integers(Spec, Min, Max, Acc) ->
 	     end,
     extract_integers(T, Min, Max, [Values|Acc]).
 
+sleep_accounting_for_max(TimeInMillis) ->
+       case (TimeInMillis > 100000000) of 
```

Comment:
Yeah, that's the best way to do it.  I found the number by calling sleep with increasingly larger numbers until it puked, then rounded it down by a couple numbers.  I'll update the code with the macro and submit another pull request.